src/patternfly/components/Button/button.scss

```diff
@@ -738,3 +746,17 @@
   display: inline-flex;
   align-items: center;
 }
+
+@keyframes #{$button}-icon-notify {
+  33% {
+    transform: rotate(30deg);
+  }
+
+  66% {
+    transform: rotate(-60deg);
+  }
+
+  100% {
+    transform: rotate(0);
+  }
```

Comment:
Just a nit but since it starts at `rotate(0)`, you shouldn't need this last part

```suggestion
```